src/GraphBLAS-sharp/Toolbox.fs

```diff
@@ -0,0 +1,98 @@
+namespace GraphBLAS.FSharp
+
+open Brahma.OpenCL
+open Brahma.FSharp.OpenCL.Core
+open Brahma.FSharp.OpenCL.Extensions
+open GlobalContext
+open Helpers
+open FSharp.Quotations.Evaluator
+open Brahma.FSharp.OpenCL.WorkflowBuilder.Basic
+open Brahma.FSharp.OpenCL.WorkflowBuilder.Evaluation
+
+type Toolbox =
```

Comment:
Should it be public?

Reply:
I guess, yes, it should. Prefix sum is useful generic helper function.

Reply:
I think, this class should be internal (like PrefixSum member), because it`s not a part of GraphBLAS public API.

Reply:
And it should be module, rather than static class.